hal/src/main/native/include/hal/HAL.h

```diff
@@ -21,7 +21,6 @@
 #include "hal/DriverStation.h"
 #include "hal/Encoder.h"
 #include "hal/Errors.h"
-#include "hal/FRCUsageReporting.h"
```

Comment:
Why are you removing this?

Reply:
It was not found when building through a wrapper

Reply:
Then you were grabbing the wrong headers. Where were you grabbing them from?

Reply:
ntcore, wpiHal, wpimath, wpinet, wpiutil

Reply:
I mean where were you actually grabbing the headers from? Those files are generated by the build, so they're not in tree.

Reply:
/ntcore /hal /wpimath /wpinet /wpiutil grabbing the whole library for definitions and just src/main/native/include for the headers to include from

Reply:
Yeah that will not work. Those headers are generated as part of the build. They're placed in differnet places depending on if you are using cmake or gradle. However, the maven header archives for each artifact do include them.

Reply:
Ok that makes sense thank you I will generate the headers using cmake then thanks for the help